### PR TITLE
Added whoami.md

### PR DESCRIPTION
**Pull request by uziemblo**

### CODE REVIEW

File: _commands/basics/whoami.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#### `whoami` vs `id -un`
+~~~ bash
+$ whoami
+uziemblo
+$ id -un
+uziemblo
+~~~
+
+##### Break it down
```
Comment (ppannuto) [inline _commands/basics/whoami.md]: Can you add an explanation here for where `id -un` came from? Why would you compare these two commands?

**Comment by ppannuto** [comment]
Could you also add a link to the page for the `id` command? You don't need to write the whole `id` page, but do make an empty stub for it.

**Comment by uziemblo** [comment]
I'm stumped what this 'external link' could be.

**Comment by uziemblo** [comment]
I Went back to my original version that worked the first time and that didn't work either. Looked at the commit history and I believe ppannuto added the Travis CI testing after I originally pushed. Any idea what the problem could be? I inspected the document, couldn't find any links. Looking at other pull requests, the only other user who has submitted in the last 5 hours is having the same issue.

**Comment by uziemblo** [comment]
For what it's worth, the empty template (that i've moved to the basics folder and renamed "id.md" is failing the automated testing as well.

**Comment by ppannuto** [comment]
Apologies, my fault. I created a chicken-and-egg problem when creating new pages with the html verifier (first time I've ever used it). Should be fixed as of 5d2e988.

**Comment by ppannuto** [comment]
I've rebased and merged this. You're all set. Thanks for the great work!

**Pull request closed without merging**